addons/mail/static/src/models/activity_mark_done_popover_view/activity_mark_done_popover_view.js

```diff
@@ -37,10 +37,11 @@ registerModel({
         /**
          * Handles click on this "Done & Schedule Next" button.
          */
-        onClickDoneAndScheduleNext() {
-            this.activityViewOwner.activity.markAsDoneAndScheduleNext({
+        async onClickDoneAndScheduleNext() {
+            await this.activityViewOwner.activity.markAsDoneAndScheduleNext({
                 feedback: this.feedbackTextareaRef.el.value,
             });
+            this.component.trigger('reload', { keepChanges: true });
```

Comment:
Note: this has been abstracted inside `chatter.reloadParentView();` now, but not all places have been adapted yet, so I understand you just copied the method above in the file, it's fine for now.